scripts/env_manager.sh

```
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Saastle – Environment Manager
+# Switch between dev/staging/prod configurations
+
+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
```

Shellcheck (0.10.0)

[warning] ROOT_DIR appears unused. Verify use (or export if used externally).

(SC2034)